docs/architecture/architecture.md

```diff
@@ -0,0 +1,169 @@
+# Algorithm Management Toolkit Architecture
+
+This document contains architectural decisions related to the Algorithm Managment Toolkit.
+
+## System Context
+
+The Algorithm Management Toolkit is an application that users can use to complete requirements
+that are specified by the algoritmekader. The diagram below sketches the broader system context
```

Comment:
I would add here also that the extra context on the other side of the the AMT is of the Algorithm Register because in the end the goal of this tool is to output a system card, this whole part i miss a bit

Reply:
Good point, I will add that to the text + to the system context diagram!